pegasus/sites.v3/code.org/public/css/educate.css

```diff
@@ -1,20 +1,24 @@
 .educate-banner {
-  margin-left: 15px;
   border-radius: 5px;
+  width: 100%;
+}
+
+#educate-curriculum-grid, #educate-resources-grid {
+  width: 100%;
+}
+
+#educate-curriculum-grid {
+  margin-top: 20px;
 }
 
 .educate-container {
```

Comment:
Just thinking preemptively for the bug bash, the https://code.org/educate/regional-partner/playbook page also uses these educate boxes. Can you do a quick look and see if it behaves the same way responsively? If we say we made these educate boxes responsive it should work in both these cases.